src/Features/Core/Portable/GenerateMember/GenerateParameterizedMember/AbstractGenerateParameterizedMemberService.SignatureInfo.cs

```diff
@@ -196,10 +197,10 @@ public ITypeSymbol DetermineReturnType(CancellationToken cancellationToken)
                     : ImmutableArray.Create(throwStatement);
             }
 
-            private ImmutableArray<IParameterSymbol> DetermineParameters(CancellationToken cancellationToken)
+            private async ValueTask<ImmutableArray<IParameterSymbol>> DetermineParametersAsync(CancellationToken cancellationToken)
             {
                 var modifiers = DetermineParameterModifiers(cancellationToken);
-                var types = DetermineParameterTypes(cancellationToken).Select(t => FixType(t, cancellationToken)).ToList();
+                var types = (await Task.WhenAll(DetermineParameterTypes(cancellationToken).Select(t => FixTypeAsync(t, cancellationToken).AsTask())).ConfigureAwait(false)).ToList();
```

Comment:
nit, extract local to make this more readable.

Reply:
Now simplified